docs/triggertemplates.md

```diff
@@ -120,3 +120,28 @@ As of Tekton Pipelines version
 embed resource specs. It is a best practice to embed each resource specs in the
 PipelineRun or TaskRun that uses the resource spec. Embedding the resource spec
 avoids a race condition between creating and using resources.
+
+
+## Quote-Escaping
```

Comment:
This primarily focuses on what the old behavior was, but doesn't really explain what the current behavior is. Let's add some more details here.

Reply:
I'm not sure what to put here, the strings are no longer escaped at all, which means that it's up to you to put the correct quoting around params if you expect the values to be quoted or not.

Reply:
One thing that could be useful is an example of what a templated JSON body would look like now and with the annotation. something like- previously or with the annotation `{\"foo\": \"bar\"}` and while now it will be `{"foo": "bar"}`